symplyphysics/laws/optics/interference_films_from_angle_refraction_refractive_index_wavelength.py

```diff
@@ -0,0 +1,57 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    angle_type
+)
+
+# Description
+## Interference in thin films is a phenomenon that occurs as a result of the separation of a light beam when
+## reflected from the upper and lower boundaries of a thin film. As a result, there are two light waves that
+## can interfere.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
```

Comment:
You can mention that order of interference can be chosen arbitrarily, to achieve the desired thin film thickness.